recipes/msmtools/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "msmtools" %}
+{% set version = "1.1.3" %}
+{% set sha256 = "0e96ea16139c9e58e710c8962ad37bb1941bac2a7827f5608eb1e9454bb9c29b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - cython >=0.24
+    - numpy x.x
+    - scipy
+    - six
+    - decorator
```

Comment:
While scipy/six/decorator are listed in the `install_requires` bit of the setup.py, the `--single-version-externally-managed` flag causes setup.py to ignore those packages.  As such, you can remove them from the build deps of the meta.yaml.